opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeMonitoredServiceRestService.java

```diff
@@ -159,6 +164,8 @@ protected Response doCreate(SecurityContext securityContext, UriInfo uriInfo, On
         final Event e = EventUtils.createNodeGainedServiceEvent("ReST", iface.getNode().getId(), iface.getIpAddress(), service.getServiceName(), iface.getNode().getLabel(),
                                                                 iface.getNode().getLabelSource(), iface.getNode().getSysName(), iface.getNode().getSysDescription());
         sendEvent(e);
+        new ApplicationEventUtil().getApplicationChangedEvents(service.getApplications()).forEach(this::sendEvent);
```

Comment:
Is there a reason why the method is not static?

Reply:
its static now.